tools/ci/tc/sink_task.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import logging
+import os
+
+import taskcluster
+
+
+logging.basicConfig()
+logger = logging.getLogger()
+
+
+def check_task_statuses(task_ids):
+    """Verifies whether a set of Taskcluster tasks completed successfully or not.
+
+    Returns 0 if all tasks passed completed successfully, 1 otherwise."""
+
+    queue = taskcluster.Queue({'rootUrl': os.environ['TASKCLUSTER_PROXY_URL']})
+    ret = 0
```

Comment:
I think I'd rather make this `success = True` and then later `return 0 if success else 1`